ambry-protocol/src/main/java/com.github.ambry.protocol/DeleteRequest.java

```diff
@@ -71,10 +125,43 @@ public boolean isSendComplete() {
     return sizeSent == sizeInBytes();
   }
 
+  public BlobId getBlobId() {
+    return blobId;
+  }
+
+  public int getCorrelationId() {
+    return correlationId;
+  }
+
+  public String getClientId() {
```

Comment:
some of these functions are already a part of the super class. why are they being redefined?